worker/worker.py

```diff
@@ -442,89 +425,115 @@ def verify_required_cutechess(cutechess_path):
                     major = int(m.group(1))
                     minor = int(m.group(2))
                     patch = int(m.group(3))
+                    short_sha = m.group(4)
     except (OSError, subprocess.SubprocessError) as e:
-        print("Unable to run cutechess-cli. Error: {}".format(str(e)))
+        print("Unable to run fast-chess. Error: {}".format(str(e)))
         return False
 
     if p.returncode != 0:
         print(
-            "Unable to run cutechess-cli. Return code: {}. Error: {}".format(
+            "Unable to run fast-chess. Return code: {}. Error: {}".format(
                 format_return_code(p.returncode), errors
             )
         )
         return False
 
     if major + minor + patch == 0:
-        print("Unable to find the version of cutechess-cli.")
+        print("Unable to find the version of fast-chess.")
         return False
 
-    if (major, minor) < (1, 2):
-        print("Requires cutechess 1.2 or higher, found version doesn't match")
+    if (major, minor) < (0, 9):
```

Comment:
lets also have these as constants at the top next around line 69

Reply:
actually, I think this check can be removed, now that we explicit check the sha.